### PR TITLE
Fix icm20948_i2c_passthrough driver that does not allow to change the address

### PR DESCRIPTION
**Pull request by roman-dvorak**
Hi, I have solved #17579. It was because the start parameters of the driver were not passed from the command line to the driver class. 

**Describe problem solved by this pull request**
Problem description is in issue #17579 

**Describe your solution**
I was inspired by other drivers where it works. The parameter for the I2C address was really missing. 

**Test data / coverage**
It was tested by connecting TFASPDIMU to CUAV V5 nano autopilot. With two different addresses.

### CODE REVIEW

File: src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp [hunk 1]
```diff
@@ -51,7 +51,7 @@ I2CSPIDriverBase *ICM20948_I2C_Passthrough::instantiate(const BusCLIArguments &c
 		int runtime_instance)
 {
 	ICM20948_I2C_Passthrough *instance = new ICM20948_I2C_Passthrough(iterator.configuredBusOption(), iterator.bus(),
-			cli.bus_frequency);
+			cli.bus_frequency, cli.i2c_address);
```
Comment (bkueng) [inline src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp:54]: `cli.rotation` needs to be passed as well if you want to make use of that.

Comment (roman-dvorak) [reply src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp:54]: It seems that IMU unit reading is not implemented in the pass-trought driver. And this parameter (rotation) is no longer used anywhere. 

So for now, I would suggest merge it like this. And leave the compass reading via I2C (in passtrought mode) alone - and in a separate PR. I can delete this adjustment from it, if it's a problem.

Comment (bkueng) [reply src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp:54]: Yes please remove it then, and also `PRINT_MODULE_USAGE_PARAM_INT('R', 0, 0, 35, "Rotation", true);`

Comment (roman-dvorak) [reply src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp:53]: Ok @bkueng, I have removed it in #d4a1802

**Review by bkueng — APPROVED** [review]

**Pull request merged**